files/build_templates/sonic_debian_extension.j2

```diff
@@ -138,11 +138,11 @@ sudo chmod a+x $FILESYSTEM_ROOT/usr/sbin/policy-rc.d
 
 {% if installer_debs.strip() -%}
 {% for deb in installer_debs.strip().split(' ') -%}
-if [ $sonic_asic_platform == "mellanox" ]; then
+{% if sonic_asic_platform == "mellanox" or sonic_asic_platform == "cavium" %}
```

Comment:
what is the issue for installation? mellanox is a workaround, it should be installation.